flang/test/Driver/use-module.f90

```diff
@@ -32,14 +32,14 @@
 
 ! INCLUDED-NOT: error
 
-! MISSING_MOD2-NOT:error: Cannot read module file for module 'basictestmoduleone''
+! MISSING_MOD2-NOT:error: Cannot parse module file for module 'basictestmoduleone''
```

Comment:
I might be missing something here, but unlike in modfile43.f90 the specific reason for not being able to parse the module file isn't given here. This is preexisting issue, hence the approval even though I am calling it out, Maybe it is the following separate error that causes this, but that isn't super obvious if it is true.

Reply:
Likewise for the following errors.

Reply:
Try compiling the test and you will observe additional informative messages.  Our message system supports messages as attachments to other messages (or attachments), and they all get formatted nicely.  For testing purposes here, however, we just care that the basic error condition has been caught.